phpcs.xml.dist

```diff
@@ -7,7 +7,7 @@
 	<exclude-pattern>/vendor/</exclude-pattern>
 	<exclude-pattern>/node_modules/</exclude-pattern>
 	<exclude-pattern>/tests/</exclude-pattern>
-	<exclude-pattern>/build/</exclude-pattern>
+	<exclude-pattern type="relative">^build/*</exclude-pattern>
```

Comment:
Devi capire a cosa serve prima di poterlo committare

Reply:
in english please

Reply:
this sets the path type to relative instead of absolute